introjs.js

```diff
@@ -0,0 +1,102 @@
+/* ========== *
+ * Challenge 1 *
+ * =========== */
+
+//Uncomment the lines below to test your code
+console.log(typeof 7.0 === "number");
+console.log(typeof 22 === "number");
+console.log(typeof true === "boolean");
+
+var myVar;
+console.log(typeof myVar === "undefined");
+
+myVar = "benny"; //add your name here
+console.log(typeof myVar === "string");
+
+//Comment them back out with two forward slashes at the beginning once you have finished
+
+/* ========== *
+ * Challenge 2 *
+ * =========== */
+
+function buildSentence(word1, word2, word3) {
+  //your code goes here...
+  let wordsOfOne = [...word1][0].toUpperCase(); // first word capital
+  let restOfwords = word1.slice(1);
+  let JoinedCapitalised = [wordsOfOne, ...restOfwords].join("");
+
+  let sentense = `${JoinedCapitalised} ${word2} ${word3}.`;
+  console.log(sentense);
+}
+
+buildSentence("coding", "is", "awesome"); //=> "Coding is awesome."
+buildSentence("we're", "number", "1"); //=> "We're number 1."
+
+/* ========== *
+ * Challenge 3 *
+ * =========== */
+
+function lastLetter(word) {
+  //your code goes here...
+  let wordLength = [...word].length;
+  let lastChar = [...word][wordLength - 1];
+  console.log(lastChar);
+}
+
+//Uncomment the lines below to test your code
+lastLetter("hello"); //=> "o"
+lastLetter("goodbye!"); //=> "!"
+lastLetter("ZeltoiD"); //=> "D"
+
+/* ========== *
+ * Challenge 4 *
+ * =========== */
+
+function buildGreeting(time, name) {
```

Comment:
@hinn254 - This should work but got a small challenge, can this be implemented using a switch statement?